FWCore/Utilities/interface/reflex.h

```diff
@@ -0,0 +1,41 @@
+#ifndef FWCore_Utilities_interface_reflex_h
```

Comment:
I wonder if this more naturally fits in FWCore/Reflection?

Reply:
Ehm, so, should I move them or not ?

Reply:
`FWCore/Reflection` depends only on `rootcling` (in addition of `FWCore/Utilities`), of which every package generating dictionaries should already depend anyway. I see `DataFormats/SoATemplate` itself does not depend on ROOT in any way, but I'd expect every use of it in conjunction with `DataFormats/Portable` to depend on 'rootcling` at minimum.

Technically this header file does not depend on ROOT, but conceptually it does (it is meant to be used in conjunction with `rootcling`), and the functionality is related to `FWCore/Reflection`  E.g. `DataFormats/Provenance` depends on `FWCore/Reflection`, so I'd expect the practical dependence impact to be minimal.

I'd be in favor of moving the header to `FWCore/Reflection`.